gateway/1.6/modules/ROOT/nav.adoc

```diff
@@ -69,7 +69,8 @@
 *** xref:policies-included-rate-limiting-sla.adoc[Rate Limiting: SLA-Based]
 *** xref:policies-included-schema-validation.adoc[Schema Validation]
 *** xref:policies-included-spike-control.adoc[Spike Control]
-*** xref:policies-included-traffic-management.adoc[Traffic Management for Multiple Upstream Services]
+*** xref:policies-included-traffic-management-route.adoc[Traffic Management for Multiple Upstream Services]
```

Comment:
Consider removing the hard-coded link text, unless you need the nav to have different text (for example, shorter).